### ISSUE TITLE
Bypass Progression for Sectors

### ISSUE THREAD

**Issue by myutin** [Anuken/Mindustry-Suggestions#3299]
### Describe the content or mechanics you are proposing.

This is the way sector progression currently works:
- you can launch to an unnamed sector from any adjacent sector
- if you have a named sector unlocked, you can launch to it from any sector on the planet
- if you don't have a named sector unlocked, you can't launch to it

Here's how I would like it to work:
- unnamed sectors are unchanged
- if you have a named sector unlocked, you can launch to it as usual
- if you don't have a named sector unlocked, you can launch to it from any adjacent sector

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

For most players, this does not change gameplay at all, as named sectors are far apart, so it is rather slow to walk down adjacent sectors. 

For speedrunners, this opens up new routing options, and encourages risk-taking (e.g. what's the lowest tech you can beat PLT with?) 

Also, if you're doing a challenge run wherein you only ever launch to adjacent sectors, it's rather annoying to pass by enemy bases that are locked behind sectors on the opposite side of the planet, having no choice but to let your sectors get captured. 

I can even imagine this with a degree of realism (e.g. there's enemy air forces that would intercept most long-range core launches, except for certain safe corridors, as per the sector progression) 

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by itcannotbe** [comment]
TL;DR: You want named sectors to be research optional?

**Comment by itcannotbe** [comment]
You do realize that those named sectors have researches locked behind them?

**Comment by myutin** [comment]
I don't see how that affects anything

a first-time player would have to clear stained mountains to unlock plastanium
(they would probably just clear all named sectors in order, to unlock the whole tech tree)

a speedrunner might figure out how to get by without plastanium, and go straight to NPC instead of SM-> FP-> NPC

**Comment by NrkSed** [comment]
You realise that plastanium/other resources can be unlocked sooner by going to unnamed sectors?

Anyway the idea isnt bad, maybe we could see such thing in the future

**Comment by myutin** [comment]
iirc the plastanium compressor is locked behind stained mountains 

if not, there's other instances of significant research locked behind certain sectors

but you're right, there is a good bit of research available without the named sectors, assuming you find the right resources in unnamed sectors

it's all part of the fun

**Comment by itcannotbe** [comment]
List of Blocks (and units) that are locked behind completing a named sector:

Launch Pad | Extraction Outpost
Interplanetary Accelerator | Planetary Terminal
Container | Biomass Facility
Vault | Stained Mountains
Pulse Conduit | Windswept Islands
Pnuematic Drill | Frozen Forest
Cultivator | Biomass Facility
Laser Drill | Nuclear Complex
Water Extractor | Salt Flats
Plastanium Compressor | Windswept Islands
Phase Weaver | Tar Fields
Kiln | Craters
Force Projector | Impact 0078
Overdrive Projector | Impact 0078
Overdrive Dome | Impact 0078
Steam Generator | Craters
Hail | Craters
Spectre | Nuclear Complex
Naval Factory | Ruinous Shores
Additive Reconstructor | Biomass Facility
Exponential Reconstructor | Overgrowth

Retusa | Wind Swept Islands
Oxynoe | Coastline
Navanax | Naval Fortress

**Comment by OnyxFinger** [comment]
lol
